### PR TITLE
Bump the Rust SDK

### PR DESCRIPTION
**Pull request by stefanceriu**
* bump the rust sdk and fix breaking changes
* cleanup where and how we present loading indicators on the app coordinator

### CODE REVIEW

File: ElementX/Sources/Services/Client/ClientProxy.swift [hunk 1]
```diff
@@ -58,10 +58,10 @@ class ClientProxy: ClientProxyProtocol {
     private var slidingSyncObserverToken: TaskHandle?
     private var slidingSync: SlidingSync?
     
-    var visibleRoomsSlidingSyncView: SlidingSyncView?
+    var visibleRoomsSlidingSyncView: SlidingSyncList?
```
Comment (pixlwave) [inline ElementX/Sources/Services/Client/ClientProxy.swift:61]: Probably not the comment you want if you're busy right now, but should we be updating all the class, variable and parameter names to match?

**Review by pixlwave — APPROVED** [review]
LGTM 👍

File: ElementX/Sources/Services/Client/ClientProxy.swift (hunk 1 above)
Comment (stefanceriu) [reply ElementX/Sources/Services/Client/ClientProxy.swift:61]: I thought about it but I honestly don't think it's worth it. If we're ever going to move this logic to the Rust side then maybe at that point.

Comment (pixlwave) [reply ElementX/Sources/Services/Client/ClientProxy.swift:61]: Very good point 👍

**Pull request merged**